src/components/performances/PerformancesSection.jsx

```diff
@@ -71,10 +74,24 @@ class PerformancesSection extends React.Component {
     // Filter function that accepts a performance object and returns true
     // if this performance passes the filter (i.e should be included)
     const filterFunction = (performance) => {
-      const danceTitle = performance.danceTitle.toLowerCase();
+      const danceTitle = performance.danceTitle ? performance.danceTitle.toLowerCase() : '';
       // Iterates through each performance metadata/key (i.e. academicLevel) and then
       // iterates through each string in the array (i.e. secondary, primary)
-      const isFilterSuccess = keys.every(key => filtersObj[key].includes(performance[key]));
+      const isFilterSuccess = keys.every((key) => {
+        // awardConsiderations need to be checked differently as it's a Number rather
+        // than a String
+        if (key === 'awardConsideration') {
+          if (filtersObj[key].includes('choreoAward')) {
+            return performance.choreoAwardEnum > 0;
+          }
+          if (filtersObj[key].includes('specialAward')) {
+            return performance.specialAwardEnum > 0;
+          }
+
+          return false;
```

Comment:
I'm curious if we should be returning false here. This is essentially a case where somehow we have a filter key on awardConsideration but none of the awards are checked. To me, I almost feel we should ignore this awardConsideration if choreoAward and specialAward are both missing.

Reply:
Just to clarify the above, lets say my filter Object looks like this:

```
{
    awardConsideration : {}
}
```

I'd argue that this award consideration is not a valid award consideration. This would actually cause *all* entries to get kicked out since no filter function on a performance will return true in this case, it would return false.

Reply:
when you mean:

> none of the awards are checked

do you mean that none of the awards have an award consideration?

Reply:
Actually nvm, I understand what you mean. I think this is actually pretty confusing so i'll explain it thoroughly. To clarify (for future reference), we have a `filterObj` that looks like this: 
```
{
  academicLevel: [],
  awardConsideration: [],
  competitionLevel: [],
  danceSize: [],
  danceStyle: [],
  school: [],
}
```

The method iterates through each key in the filterObj, when we get to `awardConsideration`, we check if the array includes `specialAward` and `choreoAward`. In the case where we have an empty array for the value of `awardsConsideration`, if there is no `choreoAward` or `specialAward` checked off to be filter (a.k.a `awardsConsideration: [] `), should we return true or false?

So yea you're right, If there are no filter values of `choreoAward` and `specialAward`, we should be returning `true`, b/c we've never set any filters on awards consideration. 

I tested it out, and technically, both work b/c:

 > The every() method tests whether all elements in the array pass the test implemented by the provided function

and this entire filter logic only gets called if whenever a filter is checked. 
 
ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every)

Reply:
This will be fine :)